### PR TITLE
ORJson. Faster Json serialization

### PR DESCRIPTION
**Pull request by michaelfeil**
## Motivation

Remove overhead in python from encoding the responses. Follow up to: https://github.com/sgl-project/sglang/pull/1688

## Modifications
Uses ORJson more wideley
```
/v1/embedding from JsonRepsponse to ORJsonResponse
```

Replaces UTF-8 encoding and f-string generation in streaming with simple byte concatenation.
```python
import orjson
import json

sample = {
    "some": "data",
    "number": 123,
    "boolean": True,
    "null": None,
    "array": [1, 2, 3],
    "object": {"key": "value"},
    "non_unicode_key": "中文",
    1: "int key",
}
# orjson returns utf-8 bytes
ex1 = b"data: " + orjson.dumps(sample, option=orjson.OPT_NON_STR_KEYS) + b"\n\n"
ex2 = f"data: {json.dumps(sample, ensure_ascii=False)}\n\n".encode("utf-8")
assert json.loads(ex1.decode("utf-8")[6:-2]) == json.loads(ex2.decode("utf-8")[6:-2])

>>> ex1
b'data: {"some":"data","number":123,"boolean":true,"null":null,"array":[1,2,3],"object":{"key":"value"},"non_unicode_key":"\xe4\xb8\xad\xe6\x96\x87","1":"int key"}\n\n'
>>> ex2
b'data: {"some": "data", "number": 123, "boolean": true, "null": null, "array": [1, 2, 3], "object": {"key": "value"}, "non_unicode_key": "\xe4\xb8\xad\xe6\x96\x87", "1": "int key"}\n\n'
```

Returning bytes (utf-8) in the generator is the same as the default StreamingResponse. https://github.com/encode/starlette/blob/46131a1af875a5be2190d91713a43ee80c8311c6/starlette/responses.py#L246

## Checklist

- [x] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ ] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [ x] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

File: python/sglang/srt/server.py [hunk 1]
```diff
@@ -429,14 +435,6 @@ def launch_server(
 
     try:
         # Listen for HTTP requests
-        LOGGING_CONFIG["formatters"]["default"][
```
Comment (zhyncs) [inline python/sglang/srt/server.py]: QQ Why did you remove the logging format code?

Comment (michaelfeil) [reply python/sglang/srt/server.py]: Sorry, missed this when pulling in the latest changes - reverted!

**Comment by zhyncs** [comment]
Hi @michaelfeil Welcome to join our slack channel https://join.slack.com/t/sgl-fru7574/shared_invite/zt-2ngly9muu-t37XiH87qvD~6rVBTkTEHw
